packages/ui-extensions-dev-console/src/theme.module.css

```diff
@@ -1,16 +1,30 @@
 .Theme {
-  /* colors */
```

Comment:
I plan to review all these colors with @MeredithCastile once we have all the functionality in place